samza-core/src/main/java/org/apache/samza/clustermanager/ContainerManager.java

```diff
@@ -136,7 +135,22 @@ boolean handleContainerLaunch(SamzaResourceRequest request, String preferredHost
         LOG.info("Waiting for running container to shutdown due to existing ContainerPlacement action {}", actionMetaData);
         return false;
       } else if (actionStatus == ContainerPlacementMetadata.ContainerStatus.STOPPED) {
-        allocator.runStreamProcessor(request, preferredHost);
+        // If the job has standby containers enabled, always check standby constraints before issuing a start on container
```

Comment:
Can the lines above 
`
if (hasActiveContainerPlacementAction(request.getProcessorId())) {
      String processorId = request.getProcessorId();	      String processorId = request.getProcessorId();
      ContainerPlacementMetadata actionMetaData = getPlacementActionMetadata(processorId).get();	      ContainerPlacementMetadata actionMetaData = getPlacementActionMetadata(processorId).get();
`

Reply:
hasActiveContainerPlacementAction checks the metadata of the action to be either in ACCEPTED or IN_PROGRESS, it not a check for the presence of metadata